flight/UAVTalk/uavtalk.c

```diff
@@ -391,7 +377,7 @@ UAVTalkRxState UAVTalkProcessInputStreamQuiet(UAVTalkConnection connectionHandle
 		}
 
 		// Check the lengths match
-		if ((iproc->rxPacketLength + iproc->instanceLength + iproc->timestampLength + iproc->length) != iproc->packet_size) { // packet error - mismatched packet size
+		if ((iproc->rxPacketLength + iproc->instanceLength + iproc->length) != iproc->packet_size) { // packet error - mismatched packet size
```

Comment:
> I don't think flight code ever receives timestamped information-- the only place timestamped information is used is in the interpretation of logs. Removing the ability for flight code to receive timestamped information seems like a good idea.

Not from us but it's a publicly documented protocol that's been around for some time. There exist a number of 3rd party implementations in the wild and they are all free to send timestamped packets of any type (perfectly valid under the protocol spec). For the sake of a few lines it makes no sense to change the protocol. There is a special place in hell for people who break their own public protocol specs IMO. Wastes lots of time trying to dig through other peoples code trying to figure out why their stuff doesn't do what it said it would do (and used to actually do) on the box.

> Bumping the version would actually break BC.

That's the point, if you make a BC change you should also change the version. How to reconcile that with the multiple other forks of this codebase is another issue that comes up when you do that.